Dockerfile.rhel8

```diff
@@ -1,6 +1,9 @@
 FROM registry.access.redhat.com/ubi8
 
 ARG GIT_COMMIT
+ARG USER_ID=1000
+ARG USER_NAME=galaxy
+ARG USER_GROUP=galaxy
```

Comment:
I am not sure if cloud deployments are using this image, so we need a review from @drodowic to check if this ok to add this defaults.

Reply:
Cloud deployments do use this image, however deployments to OpenShift will run the container with an arbitrary UID so permissions will be dependent on the group

Reply:
So is it ok to go with this change?

Reply:
I see lots of permissions errors in the devshift PR check. But it's hard to say if that's due to these changes or just because the tests are broken.
Can we get https://github.com/ansible/galaxy_ng/pull/2452 merged and then rebase this PR, so that it can be properly tested against the CRC ephemeral environment?

Reply:
https://github.com/ansible/galaxy_ng/pull/2452 has been merged. Please rebase.

Reply:
I've rebased, did some tweaks and got CI green now